Week2/homework/maartjes_work.js

```diff
@@ -45,3 +43,12 @@ const tuesday = [
 const tasks = monday.concat(tuesday);
 
 // Add your code here
+const durations = tasks
+  .map((task) => task.duration / 60)
+  .filter((task) => task >= 2);
+
+const numberOfHours = durations.reduce((accumulator, currentValue) => {
```

Comment:
As a shortcut you can use the arrow function without `return` keyword.
`durations.reduce((accumulator, currentValue) => accumulator + currentValue);`

Reply:
Ok thanks @NouranMahmoud 👍